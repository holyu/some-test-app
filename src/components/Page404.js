import React from "react";
import { Link } from "react-router-dom";

export default () => (
  <div>
    This is 404! - <Link to="/">Home</Link>
  </div>
);
