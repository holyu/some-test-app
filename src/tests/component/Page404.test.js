import React from "react";
import { shallow } from "enzyme";
import Page404 from "../../components/Page404";

test("should render Page404 correctly", () => {
  const wrapper = shallow(<Page404 />);
  expect(wrapper).toMatchSnapshot();
});
