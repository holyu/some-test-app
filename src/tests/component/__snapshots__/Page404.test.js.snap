// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Page404 correctly 1`] = `
<div>
  This is 404! - 
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
